services/alertmanager/alertmanager.go

```diff
@@ -181,6 +208,71 @@ func (svc *Service) updateConfiguration(ctx context.Context) {
 	svc.l.Infof("%s created", alertmanagerConfigPath)
 }
 
+func (svc *Service) collectRuleTemplates() {
+	rules := make([]saas.Rule, 0)
```

Comment:
Can be `var rules []saas.Rule`, but it's up to you.

Reply:
Oh, I see linter warning
https://github.com/percona/pmm-managed/pull/525#discussion_r526421040

In that case, you can move that line down, where you know `shippedFilePaths` size, and preallocate `rules` with the proper length.